### ISSUE TITLE
Medium.en model just outputting "Okay" for every second in the audio while the base.en model works well

### ISSUE THREAD

**Issue by bangpradyumna** [ggerganov/whisper.cpp#719]
Hello Everyone,
I have a recording that I'm trying to transcribe. I first tried doing that using base model which worked fine but not perfect. I then tried doing the same using the Medium.en model but it just outputs "Okay" for each second of the audio. 

Although there are 5 or 6 "Okays" in the audio but Medium model just keeps on outputting "Okay" even for lines which the "Base" model is able to transcribe.

Screenshot of Base.en model's output which works well : 
![image](https://user-images.githubusercontent.com/6296142/230065749-a2290852-c585-42a2-881f-5b2bafe7a152.png)

Screenshot of Medium.en model's output  :
![image](https://user-images.githubusercontent.com/6296142/230065165-ef02eeb6-0be8-4274-b5de-7154f0927601.png)

Any idea on what I might be doing wrong ?

**Comment by carlosbaraza** [comment]
I having the same problem:

```
[00:37:36.000 --> 00:38:02.000]   >> Thank you.
[00:38:02.000 --> 00:38:28.000]   >> Thank you.
[00:38:28.000 --> 00:38:54.000]   >> Thank you.
[00:38:54.000 --> 00:39:20.000]   >> Thank you.
[00:39:20.000 --> 00:39:46.000]   >> Thank you.
[00:39:46.000 --> 00:40:12.000]   >> Thank you.
[00:40:12.000 --> 00:40:38.000]   >> Thank you.
[00:40:38.000 --> 00:41:04.000]   >> Thank you.
[00:41:04.000 --> 00:41:30.000]   >> Thank you.
[00:41:30.000 --> 00:41:56.000]   >> Thank you.
[00:41:56.000 --> 00:42:22.000]   >> Thank you.
[00:42:22.000 --> 00:42:48.000]   >> Thank you.
[00:42:48.000 --> 00:43:14.000]   >> Thank you.
[00:43:14.000 --> 00:43:40.000]   >> Thank you.
[00:43:40.000 --> 00:44:06.000]   >> Thank you.
[00:44:06.000 --> 00:44:32.000]   >> Thank you.
[00:44:32.000 --> 00:44:58.000]   >> Thank you.
[00:44:58.000 --> 00:45:24.000]   >> Thank you.
[00:45:24.000 --> 00:45:50.000]   >> Thank you.
[00:45:50.000 --> 00:46:16.000]   >> Thank you.
[00:46:16.000 --> 00:46:42.000]   >> Thank you.
[00:46:42.000 --> 00:47:08.000]   >> Thank you.
[00:47:08.000 --> 00:47:34.000]   >> Thank you.
[00:47:34.000 --> 00:48:00.000]   >> Thank you.
[00:48:00.000 --> 00:48:26.000]   >> Thank you.
[00:48:26.000 --> 00:48:52.000]   >> Thank you.
[00:48:52.000 --> 00:49:12.000]   >> Thank you.
[00:49:12.000 --> 00:49:38.000]   >> Thank you.
[00:49:38.000 --> 00:50:04.000]   >> Thank you.
[00:50:04.000 --> 00:50:30.000]   >> Thank you.
[00:50:30.000 --> 00:50:56.000]   >> Thank you.
[00:50:56.000 --> 00:51:22.000]   >> Thank you.
[00:51:22.000 --> 00:51:42.000]   >> Thank you.
[00:51:42.000 --> 00:52:08.000]   >> Thank you.
[00:52:08.000 --> 00:52:34.000]   >> Thank you.
[00:52:34.000 --> 00:53:00.000]   >> Thank you.
[00:53:00.000 --> 00:53:26.000]   >> Thank you.
[00:53:26.000 --> 00:53:52.000]   >> Thank you.
[00:53:52.000 --> 00:54:12.000]   >> Thank you.
[00:54:12.000 --> 00:54:38.000]   >> Thank you.
[00:54:38.000 --> 00:55:04.000]   >> Thank you.
[00:55:04.000 --> 00:55:30.000]   >> Thank you.
[00:55:30.000 --> 00:55:56.000]   >> Thank you.
[00:55:56.000 --> 00:56:22.000]   >> Thank you.
[00:56:22.000 --> 00:56:48.000]   >> Thank you.
[00:56:48.000 --> 00:57:14.000]   >> Thank you.
[00:57:14.000 --> 00:57:40.000]   >> Thank you.
[00:57:40.000 --> 00:58:06.000]   >> Thank you.
[00:58:06.000 --> 00:58:32.000]   >> Thank you.
[00:58:32.000 --> 00:58:58.000]   >> Thank you.
[00:58:58.000 --> 00:59:24.000]   >> Thank you.
[00:59:24.000 --> 00:59:50.000]   >> Thank you.
[00:59:50.000 --> 01:00:16.000]   >> Thank you.
[01:00:16.000 --> 01:00:42.000]   >> Thank you.
[01:00:42.000 --> 01:01:08.000]   >> Thank you.
[01:01:08.000 --> 01:01:34.000]   >> Thank you.
[01:01:34.000 --> 01:02:00.000]   >> Thank you.
[01:02:00.000 --> 01:02:26.000]   >> Thank you.
[01:02:26.000 --> 01:02:52.000]   >> Thank you.
[01:02:52.000 --> 01:03:14.000]   >> Thank you.
[01:03:14.000 --> 01:03:40.000]   >> Thank you.
[01:03:40.000 --> 01:04:06.000]   >> Thank you.
[01:04:06.000 --> 01:04:32.000]   >> Thank you.
[01:04:32.000 --> 01:04:52.000]   >> Thank you.
```

**Comment by abelbabel** [comment]
I have the same issue ... seems not to be related to a specific model ... and not with each input file ...

**Comment by ggerganov** [comment]
I've disabled the decoder fallbacks because current implementation is very inefficient.
This will be resolved some time in the future

**Comment by abelbabel** [comment]
Turned out that in one case the section where multiple "Okay"s were "hallucinated" was loud rumbling / noises (no speech). I isolated this part and it was detected correctly. After that I took one detected noise output (like "(pages rustling)") as an input for the prompt-parameter and the original file was detected properly.

This is of course not working in large scale.
But maybe it gives an idea where the problem is ...
